content/applications/sales/sales/amazon_connector/update.rst

```diff
@@ -55,18 +48,14 @@ Install the Amazon/Authentication Patch
     you should use a staging environment - you should contact your integrating partner for
```

Comment:
Editing wording and punctuation.
```suggestion
    you should use a staging environment--contact your integrating partner for
```

Reply:
I understand why we should take off `you should` here, but I'm not too sure about the replacement, is it something I'm not english-speakery enough to know about? :sweat_smile:  I'd have used `:` instead, I think. I put your suggestion anyway, but keep me updated so that I can change it if necessary (or learned an interesting english uses of the punctuation :smile: ). 

(BTW, since we are changing things in this paragraph, I fixed the indentation as well, so don't mind the huge diff, I only changed what you told me to ^^)

Reply:
Ho, nice, indeed, `---` did the trick :partying_face: